### PR TITLE
Enable esModuleInterop for better type checking support

### PR DESCRIPTION
**Pull request by Stanzilla**
Motivation: ¯\_(ツ)_/¯

### CODE REVIEW

File: webpack.config.ts [hunk 1]
```diff
@@ -224,7 +224,7 @@ export default (_env: never, args: Configuration): Configuration[] => {
         ...args.watch ? [] : [
 
           /* FontAwesome icons */
-          new CopyPlugin({
+          new CopyPlugin.default({
```
Comment (squidfunk) [inline webpack.config.ts:227]: That is no idiomatic JS/TS 😅

**Review by squidfunk — CHANGES_REQUESTED** [review]

**Comment by squidfunk** [comment]
Why would we want `esModuleInterop` to be activated? It's meant as a last resort when working with code that would not work otherwise. Everything is working as expected. I use it very sparingly in my projects.

**Comment by Stanzilla** [comment]
Just for type checking, really. I mostly played around with it and figured why not, hence the PR note being a shruggy :)

**Comment by squidfunk** [comment]
Well, as noted I'd rather not enable `esModuleInterop` as it may increase the bundle size unless absolutely necessary.

**Pull request closed without merging**